Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -1076,8 +1076,21 @@ module.exports = {
       .setFunctionName('setLinearVelocityX')
       .setManipulatedType('number')
       .setGetter('getLinearVelocityX');
-
-    aut
+	
+	  aut
```

Comment:
No, you had to *remove* two spaces before aut, not add them ;)

Reply:
I have vscode and installed prettier, says installed and shows on status bar.  But I press CTRL+Shift+P and nothing happens :(